athena-federation-integ-test/src/main/java/com/amazonaws/athena/connector/integ/data/ConnectorPackagingAttributes.java

```diff
@@ -0,0 +1,64 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation Integ Test
+ * %%
+ * Copyright (C) 2019 - 2020 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connector.integ.data;
+
+/**
+ * Contains the the connector's packaging attributes.
+ */
+public class ConnectorPackagingAttributes
+{
+    private final String s3Bucket;
+    private final String s3Key;
+    private final String lambdaFunctionHandler;
```

Comment:
comments to describe what these are used for? a javadoc that is basically the name of the class isn't super useful for future maintainers.

Reply:
Added detailed description.